tests/test_dummy.py

```diff
@@ -2,5 +2,6 @@
 
 from pyannote.database import get_databases
 
+
```

Comment:
Looks like change in `tests/test_dummy.py` are unrelated to this PR main topic.
Can you revert those changes?